robots/dissemination/cleanup.rb

```diff
@@ -26,6 +26,10 @@ def is_full_druid_tree(druid)
           truncate_druid_tree = DruidTools::AccessDruid.new(druid, workspace_root)
           (!Dir.glob(truncate_druid_tree.path).empty? && !Dir.glob(full_druid_tree.path + '*').empty?)
         end
+
+        def workflow_service
```

Comment:
I like the clarity of `Robots::DorRepo::Accession::Base` and it's a little confusing as to why the cleanup class has the new code, whereas it makes more sense in a Base class. If we don't put it in a Base class, then maybe a comment as to why it's in the cleanup class (i.e., "move this into a base class if we get more dissemination robots").